src/platform/forms-system/src/js/review/ObjectField.jsx

```diff
@@ -162,14 +162,7 @@ class ObjectField extends React.Component {
       onEdit = formContext?.onEdit,
       text = 'Edit',
     } = {}) => (
-      <button
-        type="button"
-        className="edit-btn primary-outline"
-        aria-label={label}
-        onClick={onEdit}
-      >
-        {text}
-      </button>
+      <va-button secondary aria-label={label} onClick={onEdit} text={text} />
```

Comment:
This is for the part of the ticket about the edit button accessibility. This and the line on 190 have to do with that. The rest of the PR is for the accordions.

Reply:
This class name still needs to be added. As I mentioned, some [apps & tests use this class name](https://github.com/search?q=repo%3Adepartment-of-veterans-affairs%2Fvets-website%20.edit-btn&type=code)

Reply:
Weird, I remember doing this...

Reply:
Oh, you know what, no, I tried it but there's no way to add class names to `va-button`.

I think we'll have to go look at each case of its use individually and see if it's referring to this page or another page (since the class 'edit-btn' is used elsewhere as well) and fix it if it's referring to this page.

Reply:
Ok so I don't think any of these are related:

```
👾  vets-website(dbex-64149-button-contrast)± $ rg '\.edit-btn'                                                                       
src/applications/financial-status-report/sass/financial-status-report.scss
356:@include hide-button(".edit-btn.primary-outline");

src/applications/appeals/testing/components/AreaOfDisagreement.jsx
107:        `[name="areaOfDisagreementFollowUp${pagePerItemIndex}ScrollElement"] + form .edit-btn`,

src/applications/appeals/shared/components/AreaOfDisagreement.jsx
106:        `[name="areaOfDisagreementFollowUp${pagePerItemIndex}ScrollElement"] + form .edit-btn`,

src/applications/edu-benefits/10203/tests/edu-10203.cypress.spec.js
79:        cy.get('.edit-btn').click({ multiple: true });

src/applications/hca/tests/components/FormReview/DependentsReviewPage.unit.spec.js
22:      const selector = container.querySelector('.edit-btn');
55:      const selector = container.querySelector('.edit-btn');
85:      const selector = container.querySelector('.edit-btn');

src/platform/forms-system/src/js/review/ArrayField.jsx
219:      this.scrollToAndFocus(`${fieldName}-${index}`, '.edit-btn');
```

`financial-status-report.scss`: is scoped to `.form-review-panel`, of which there is only one on the entire site, and it's not on this page. In fact, it's on another custom accordion that will need to be replaced some day. I do wonder if it's a typo, because there are dozen if not hundreds of `'.form-review-panel-page'`, but that's so far out of scope it's not something I'm going to worry about!

`AreaOfDisagreement.jsx` (both of them): Yep, fixed these!

`edu-10203.cypress.spec.js`: This is fine since it's a passing cypress test. If this was referring to the `<va-button>` as opposed to some other button with the same class then the `.click` would throw an exception.

`DependentsReviewPage.unit.spec.js`: Also a passing test. All of these queries are explicitly tested as existing.

`ArrayField.jsx`: This refers to some code that refocuses the edit button after editing an array element. This is a feature of the review page and is working, so this is not being used here, and is therefor referring to some other page.

This `ObjectField.jsx` is _only_ imported by `SchemaForm.jsx` (plus some tests) for review fields, so I don't believe we'll see it popping up on pages other than this review page, which means the scope is quite limited.

I know this is a bit all over the place, let me know what you think! I'm pretty confident in it.

Reply:
Good morning @Mottie, how are you feeling about this?